src/pages/myAdmin/guides/concepts.tsx

```diff
@@ -348,8 +348,7 @@ export default function Concepts() {
                         JSON
                     </a>{" "}
                     format. The{" "}
-                    {/* TODO: update link */}
-                    <Link to=""> Reference</Link>{" "}
+                    <Link to="./../../apiReference/methods"> Reference</Link>{" "}
```

Comment:
not the right path

Reply:
Has been updated to "/myAdmin/apiReference/methods"